packages/module/patternfly-docs/generated/extensions/user-feedback/design-guidelines.js

```diff
@@ -27,14 +27,14 @@ pageData.examples = {
 
 const Component = () => (
   <React.Fragment>
-    <p {...{"className":"ws-p"}}>
+    <p {...{"className":"undefinedpf-v6-c-content--p ws-p "}}>
```

Comment:
Can you bump the docs framework version to make sure these `undefined`s go away?